<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RandomDecisionForestDialogGui</class>
 <widget class="QDialog" name="RandomDecisionForestDialogGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>839</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QGroupBox" name="gbParameters">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Parameters</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <property name="spacing">
       <number>1</number>
      </property>
      <item row="1" column="1">
       <widget class="QSpinBox" name="spinBox_NTrees">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Number of trees:&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;Bla bla.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="value">
         <number>12</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_NTrees">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Number of Trees</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_ProbX_2">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Probe Distance X</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="spinBox_probX">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="value">
         <number>30</number>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QSpinBox" name="spinBox_MinLeafPixels">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="value">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_MaxDepth">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Max Depth</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="spinBox_MaxDepth">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="value">
         <number>20</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_PixelsPerImage">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Pixels per Image</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="label_MaxIteration">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Max Iteration</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QSpinBox" name="spinBox_PixelsPerImage">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="maximum">
         <number>300</number>
        </property>
        <property name="value">
         <number>200</number>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_MinLeafPixels">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Min Leaf Pixels</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QSpinBox" name="spinBox_MaxIteration">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_ProbY_2">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Probe Distance Y</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="spinBox_probY">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="value">
         <number>30</number>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="label_LabelCount">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Label Count</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QSpinBox" name="spinBox_LabelCount">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="value">
         <number>26</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Training</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_train">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <property name="spacing">
       <number>1</number>
      </property>
      <item row="1" column="0" colspan="4">
       <widget class="QTextBrowser" name="textBrowser_train"/>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="loadTrainData_button">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="loadForest_button">
        <property name="text">
         <string>Load</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="train_button">
        <property name="text">
         <string>Train</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="saveForest_button">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Test</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_test">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <property name="spacing">
       <number>1</number>
      </property>
      <item row="0" column="0">
       <widget class="QPushButton" name="loadTestData_button">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="test_button">
        <property name="text">
         <string>Test</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QTextBrowser" name="textBrowser_test"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_NTrees_2">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Number of Trees</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="spinBox_NTestTrees">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Number of trees:&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;Bla bla.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>spinBox_NTrees</tabstop>
  <tabstop>spinBox_MaxDepth</tabstop>
  <tabstop>spinBox_probX</tabstop>
  <tabstop>spinBox_probY</tabstop>
  <tabstop>spinBox_PixelsPerImage</tabstop>
  <tabstop>spinBox_MinLeafPixels</tabstop>
  <tabstop>spinBox_MaxIteration</tabstop>
  <tabstop>spinBox_LabelCount</tabstop>
  <tabstop>loadTrainData_button</tabstop>
  <tabstop>textBrowser_train</tabstop>
  <tabstop>loadTestData_button</tabstop>
  <tabstop>test_button</tabstop>
  <tabstop>textBrowser_test</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>train_button</sender>
   <signal>clicked()</signal>
   <receiver>RandomDecisionForestDialogGui</receiver>
   <slot>onTrain()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>307</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>524</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loadTrainData_button</sender>
   <signal>clicked()</signal>
   <receiver>RandomDecisionForestDialogGui</receiver>
   <slot>onTrainingBrowse()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>646</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loadTestData_button</sender>
   <signal>clicked()</signal>
   <receiver>RandomDecisionForestDialogGui</receiver>
   <slot>onTestBrowse()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>519</y>
    </hint>
    <hint type="destinationlabel">
     <x>573</x>
     <y>507</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>test_button</sender>
   <signal>clicked()</signal>
   <receiver>RandomDecisionForestDialogGui</receiver>
   <slot>onTest()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>307</x>
     <y>519</y>
    </hint>
    <hint type="destinationlabel">
     <x>561</x>
     <y>491</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loadForest_button</sender>
   <signal>clicked()</signal>
   <receiver>RandomDecisionForestDialogGui</receiver>
   <slot>onLoad()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>187</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel">
     <x>533</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveForest_button</sender>
   <signal>clicked()</signal>
   <receiver>RandomDecisionForestDialogGui</receiver>
   <slot>onSave()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>357</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>609</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_NTrees</sender>
   <signal>valueChanged(int)</signal>
   <receiver>RandomDecisionForestDialogGui</receiver>
   <slot>onNTreesChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>181</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>201</x>
     <y>-71</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_MaxDepth</sender>
   <signal>valueChanged(int)</signal>
   <receiver>RandomDecisionForestDialogGui</receiver>
   <slot>onMaxDepthChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>166</x>
     <y>-57</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_probX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>RandomDecisionForestDialogGui</receiver>
   <slot>onProbDistXChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>-16</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_probY</sender>
   <signal>valueChanged(int)</signal>
   <receiver>RandomDecisionForestDialogGui</receiver>
   <slot>onProbDistYChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>156</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>-20</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_PixelsPerImage</sender>
   <signal>valueChanged(int)</signal>
   <receiver>RandomDecisionForestDialogGui</receiver>
   <slot>onPixelsPerImageChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>365</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>531</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_MinLeafPixels</sender>
   <signal>valueChanged(int)</signal>
   <receiver>RandomDecisionForestDialogGui</receiver>
   <slot>onMinLeafPixelsChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>377</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>590</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_MaxIteration</sender>
   <signal>valueChanged(int)</signal>
   <receiver>RandomDecisionForestDialogGui</receiver>
   <slot>onMaxIterationChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>383</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>587</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_LabelCount</sender>
   <signal>valueChanged(int)</signal>
   <receiver>RandomDecisionForestDialogGui</receiver>
   <slot>onLabelCountChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>374</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>565</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_NTestTrees</sender>
   <signal>valueChanged(int)</signal>
   <receiver>RandomDecisionForestDialogGui</receiver>
   <slot>onNTestTreesChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>560</y>
    </hint>
    <hint type="destinationlabel">
     <x>575</x>
     <y>547</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onTrain()</slot>
  <slot>onTrainingBrowse()</slot>
  <slot>onTestBrowse()</slot>
  <slot>onTest()</slot>
  <slot>onLoad()</slot>
  <slot>onSave()</slot>
  <slot>onNTreesChanged(int)</slot>
  <slot>onMaxDepthChanged(int)</slot>
  <slot>onProbDistXChanged(int)</slot>
  <slot>onProbDistYChanged(int)</slot>
  <slot>onPixelsPerImageChanged(int)</slot>
  <slot>onMinLeafPixelsChanged(int)</slot>
  <slot>onMaxIterationChanged(int)</slot>
  <slot>onLabelCountChanged(int)</slot>
  <slot>onNTestTreesChanged(int)</slot>
 </slots>
</ui>
